development/readme.md

```diff
@@ -9,11 +9,8 @@ There are images for each base OS:
 - [`mantidproject/mantid-development-centos7`](https://hub.docker.com/r/mantidproject/mantid-development-centos7/) - CentOS 7
 - [`mantidproject/mantid-development-ubuntubionic`](https://hub.docker.com/r/mantidproject/mantid-development-ubuntubionic/) - Ubuntu 18.04 (Bionic)
 
-Typically you'd want to use the `latest` tag.
```

Comment:
Can we keep the sentence about "Typically you'd want to use the `latest` tag."?

Reply:
Actually the next one too, just removing the bit about paraview from the sentence.

Reply:
Done!